test/suite/commands/edit-align.md

```diff
@@ -0,0 +1,122 @@
+# 1
+
+```
+foo:bar
+   ^ 0
+longfoo:bar
+       ^ 1
+foo2:longbar
+    ^ 2
+```
+
+## 1 align
+[up](#1)
+
+- .edit.align
+
+```
+foo    :bar
+       ^ 0
+longfoo:bar
+       ^ 1
+foo2   :longbar
+       ^ 2
+```
+
+
+# 2
+
+```
+selections are aligned by their selection active point
+ --> 97) lorem
+     ^^^ 0
+ --> 98) ipsum
+     ^^^ 1
+ --> 99) dolor
+     ^^^ 2
+ --> 100) sit
+     ^^^^ 3
+ --> 101) amet
+     ^^^^ 4
+```
+
+## 2 align
```

Comment:
I'd like to have a test that also shows what happens when selections are reversed (so nothing changes), to further display the behavior.

Reply:
Ha, never mind, you did this below, thanks!